packages/storage/src/implementation/requests.ts

```diff
@@ -163,7 +163,7 @@ export function list(
   pageToken?: string | null,
   maxResults?: number | null
 ): RequestInfo<ListResult> {
-  let urlParams = {};
+  let urlParams: { [key: string]: any } = {};
```

Comment:
We have another PR pending that removes `any` from Storage. Can you use `unknown` here?

Reply:
I actually just saw that there was a `UrlParams` type so I used that.  The only change is I have to explicitly cast `maxResults` to string if that's okay.

Reply:
Oops I guess it wasn't okay.  I changed UrlParams to accept `string | number` values, that seems like a better idea.